contrib/cross-compile-armv6hf/Dockerfile

```diff
@@ -11,20 +11,18 @@ FROM --platform=linux/amd64 ubuntu:18.04
 
 # Install common packages.
 RUN apt-get update
-RUN apt-get install -y -qq git curl build-essential libasound2-dev libssl-dev libpulse-dev libdbus-1-dev
+RUN apt-get install -y -qq git curl build-essential cmake clang libclang-dev libasound2-dev libpulse-dev
 
 # Install armhf packages.
 RUN echo "deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ bionic main" | tee -a /etc/apt/sources.list
 RUN apt-get update
-RUN apt-get download libasound2:armhf libasound2-dev:armhf libssl-dev:armhf libssl1.1:armhf
+RUN apt-get download libasound2:armhf libasound2-dev:armhf
 RUN mkdir /sysroot && \
     dpkg -x libasound2_*.deb /sysroot/ && \
-    dpkg -x libssl-dev*.deb /sysroot/ && \
-    dpkg -x libssl1.1*.deb /sysroot/ && \
     dpkg -x libasound2-dev*.deb /sysroot/
 
 # Install rust.
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.81 -y
```

Comment:
Is this required? Or what are the intentions behind this change?

Reply:
Strictly - not required. I wanted to make rust version explicit and aligned with MSRV,

Reply:
Make sense, just wanted to know if there was a deeper meaning behind it. For example missing comparability in stable or so.